tools/monorepo-utils/src/workflow-profiler/commands/profile/index.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import { Command } from '@commander-js/extra-typings';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getWorkflowRunData,
+	logWorkflowRunResults,
+	getWorkflowData,
+} from '../../lib';
+import { Logger } from '../../../core/logger';
+import config from '../../config';
+
+const program = new Command( 'profile' )
+	.description( 'Profile Github workflows' )
+	.argument( '<start>', 'Start date in YYYY-MM-DD format' )
+	.argument( '<end>', 'End date in YYYY-MM-DD format' )
+	.argument( '[ids...]', 'Workflow IDs. Defaults to required workflows' )
```

Comment:
```suggestion
	.argument( '[ids...]', 'Workflow IDs. Defaults to default workflows' )
```

Just in case my suspicion further down is correct.